### PR TITLE
Documentation: Fixes broken links and standardizes headers.

### PR DESCRIPTION
**Pull request by intelkevinputnam**
As requested, header and link changes broken into separate PR. Based on PR: #446 

Signed-off-by: Kevin Putnam <kevin.putnam@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 [![Go Report Card](https://goreportcard.com/badge/github.com/intel/intel-device-plugins-for-kubernetes)](https://goreportcard.com/report/github.com/intel/intel-device-plugins-for-kubernetes)
 [![GoDoc](https://godoc.org/github.com/intel/intel-device-plugins-for-kubernetes/pkg/deviceplugin?status.svg)](https://godoc.org/github.com/intel/intel-device-plugins-for-kubernetes/pkg/deviceplugin)
 
+This repository contains a framework for developing plugins for the Kubernetes
+[device plugins framework](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/),
+along with a number of device plugin implementations utilising that framework.
+
 ## Table of Contents
 
 * [About](#about)
```
Comment (linjiach) [inline README.md:12]: Can we remove this #about since the About section below is removed to be Overview?

**Review by mythi — COMMENTED** [review]
LGTM!

**Review by bart0sh — DISMISSED** [review]
/lgtm

File: cmd/fpga_crihook/README.md [hunk 2]
```diff
@@ -9,27 +9,27 @@
...
 The CRI-O prestart hook is only *required* when the
-[FPGA admission webhook](../fpga_admissionwebhook) is configured for orchestration
+[FPGA admission webhook](../fpga_admissionwebhook/README.md) is configured for orchestration
 programmed mode, and is benign (un-used) otherwise.
 
 > **Note:** The fpga CRI-O webhook is usually installed by the same DaemonSet as the
 > FPGA device plugin. If building and installing the CRI-O webhook by hand, it is
 > recommended you reference the
-> [fpga plugin DaemonSet YAML](../../deployments/fpga_plugin/fpga_plugin.yaml) for
+> [fpga plugin DaemonSet YAML](/deployments/fpga_plugin/fpga_plugin.yaml) for
```
Comment (rojkov) [inline cmd/fpga_crihook/README.md:29]: Is this link really broken? Would it be safer as an absolute path? I'm a bit concerned how it's going to be handled by HTML renderers: would it be always treated as absolute inside the repo, not the website (e.g. https://github.com/deployments/...).

Comment (intelkevinputnam) [reply cmd/fpga_crihook/README.md:29]: In this case, that file doesn't exist in that location and doesn't work in the existing documentation. Absolute paths work for both GitHub and Sphinx. The sphinx-md extension can either upload the files with the docs or link to the source commit in GitHub. I prefer the source commit route, which is configurable.

File: README.md (hunk 1 above)
Comment (intelkevinputnam) [reply README.md:12]: Good idea.

Comment (linjiach) [reply README.md]: looking good now!

File: cmd/fpga_crihook/README.md (hunk 2 above)
Comment (mythi) [reply cmd/fpga_crihook/README.md:29]: there's only `deployments/fpga_plugin/base/intel-fpga-plugin-daemonset.yaml`

Comment (rojkov) [reply cmd/fpga_crihook/README.md:29]: Oh, right. The file simply doesn't exist in that path in the master branch. Then what appears in the generated docs then?

**Comment by mythi** [comment]
@intelkevinputnam can you rebase this? Another PR updated some parts of this PR so we're seeing conflicts

**Comment by intelkevinputnam** [comment]
@mythi Will do.

File: cmd/fpga_plugin/README.md [hunk 3]
```diff
@@ -136,7 +136,7 @@ major components:
 -   [FPGA prestart CRI-O hook](../fpga_crihook/README.md)
 
 The CRI-O hook is only *required* if `region` mode is being used, but is installed by default by the
-[FPGA plugin DaemonSet YAML](../../deployments/fpga_plugin/fpga_plugin.yaml), and is benign
+[FPGA plugin DaemonSet YAML](/deployments/fpga_plugin/fpga_plugin.yaml), and is benign
```
Comment (mythi) [inline cmd/fpga_plugin/README.md:139]: `/deployments/fpga_plugin/base/intel-fpga-plugin-daemonset.yaml`?

File: cmd/fpga_crihook/README.md [hunk 4]
```diff
@@ -29,7 +29,7 @@ programmed mode, and is benign (un-used) otherwise.
 > [fpga plugin DaemonSet YAML](../../deployments/fpga_plugin/base/intel-fpga-plugin-daemonset.yaml) for
```
Comment (mythi) [inline cmd/fpga_crihook/README.md:29]: `/deployments/fpga_plugin/base/intel-fpga-plugin-daemonset.yaml`?

**Review by mythi — CHANGES_REQUESTED** [review]
please squash the commits to one

**Comment by intelkevinputnam** [comment]
@mythi Done.

**Review by mythi — APPROVED** [review]

**Review by rojkov — APPROVED** [review]

**Comment by mythi** [comment]
@intelkevinputnam thanks! Let's revisit the Sphinx PR next?

**Pull request merged**